src/Workspaces/Core/Portable/Rename/Renamer.cs

```diff
@@ -23,6 +26,64 @@ public static class Renamer
                 newName, optionSet, cancellationToken);
         }
 
+        /// <summary>
+        /// Call to perform a rename of document or change in document folders. Performs additional code changes related to the document
+        /// being renamed or moved. 
```

Comment:
instead of `performs additional` it should probably be something more like `Returns additional code changes that are related to the document being renamed (for example, renaming symbols contained within the file to match the new file name).